src/lib/apiTypesV2/SubscriptionUpdate.h

```diff
@@ -32,23 +32,26 @@ namespace ngsiv2
   class SubscriptionUpdate : public Subscription
   {
   public:
-    bool  descriptionProvided;
     bool  subjectProvided;
     bool  expiresProvided;
     bool  statusProvided;
     bool  notificationProvided;
     bool  attrsFormatProvided;
     bool  throttlingProvided;
 
+    bool  fromNgsiv1;          // to support an special case when the SubscriptionUpdate comes from NGSIv1
+
     SubscriptionUpdate():
-      descriptionProvided  (false),
       subjectProvided      (false),
       expiresProvided      (false),
       statusProvided       (false),
       notificationProvided (false),
       attrsFormatProvided  (false),
-      throttlingProvided   (false)
-      {}
+      throttlingProvided   (false),
+      fromNgsiv1           (false)
+      {
+        descriptionProvided = false;
+      }
```

Comment:
It has been removed from the child class (so I cannot use initialization list), but stills in the parent class.

I mean, descriptionProvided was (wrongly) being used in both chil and parent. I have removed in child, but I cannot use initialization list now (or maybe is possible but I don't know how :)